pom.xml

```diff
@@ -49,7 +49,7 @@
   <parent>
     <groupId>org.jahia.modules</groupId>
     <artifactId>jahia-modules</artifactId>
-    <version>8.2.0.0</version>
```

Comment:
Look like this bump is not required anymore, since you manage to get all the code inside the module it self :) ?

Reply:
you're right! Good catch, I forgot to revert it, thanks 🙏